src/dmd/link.d

```diff
@@ -34,6 +35,23 @@ version (Posix) extern (C) int pipe(int*);
 version (Windows) extern (C) int spawnlp(int, const char*, const char*, const char*, const char*);
 version (Windows) extern (C) int spawnl(int, const char*, const char*, const char*, const char*);
 version (Windows) extern (C) int spawnv(int, const char*, const char**);
+version (Posix) extern (C) extern __gshared char** environ;
```

Comment:
Why is this not in the `CRuntime_Glibc` block, since it's only used there ?

Reply:
I didn't want to imply it was glibc. environ exists on every POSIX system.